index.qmd

```diff
@@ -45,11 +45,96 @@ my-custom-stuff:
 
 ## Contents
 
+* Hardware overview
 * Introduction to High Performance Computing
 * SWC HPC system
 * Using the job scheduler
 * Running pose estimation on the SWC HPC
 
+## Hardware overview {.smaller}
+::: {.fragment}
+* CPU (Central Processing Unit)
+    * General-purpose
+    * Split into cores (typically between 4 and 64)
+    * Each core can run a separate process
+    * Typically higher clock speed than GPU (~3-5GHz)
+:::
+::: {.fragment}
+* GPU (Graphics Processing Unit)
+    * Originally for rendering graphics
+    * Thousands of cores
+    * Optimised for parallel processing of matrix multiplication
+    * Typically lower clock speed than CPU (~1-2GHz)
+:::
+
+## Hardware overview {.smaller}
+#### Primary storage:
+
+::: {.fragment}
+* Cache
+    * Small, fast memory
+    * Stores frequently accessed data
+    * Fastest outside of CPU/GPU registers
+    * Typically in the MB range with multiple levels
+:::
+::: {.fragment}
+* Main memory (RAM/VRAM)
+    * Fast storage for data
+    * CPU/GPU can access data quickly
+    * Lost when machine is powered off
+    * Typically 8-512 GB range
+:::
+
+## Hardware overview {.smaller}
+#### Secondary storage:
+
+::: {.fragment}
+* Drive storage (HDD/SSD)
+    * Much slower than RAM
+    * SSDs faster than HDDs
+    * Typically in the GB-TB range
+:::
+::: {.fragment}
+* Network storage (ceph)
```

Comment:
At SWC we tend to refer to ""ceph" as the word for "big storage". It would be good to try and enforce that ceph is just one technology, and we happen to have an instance of it, also (confusingly) called "ceph". 
```suggestion
* Network storage (e.g. ceph)
```